### ISSUE TITLE
How to set windowsize and threshold

### ISSUE THREAD

**Issue by chenfeiyi** [crazyzxd/surface-normal#2]
Thank you for sharing your code.

After using this tool on my own dataset,  the result seems to be abnormal, and I think the value of windowsize and threshold is not suitable. I haven't fully understood this code, can you tell me how to set the value of windowsize and threshold? Additionally, fcxcy is the intrinsic parameters 
k=[fx,0,u0,
     0,fy,v0,
     0, 0, 0]   which means fcxcy[0]= fx= fx. fcxcy[1] = u0, fcxcy[2] =v0, is that right?

**Comment by Cindy-xdZhang** [comment]
Yes, you are right. In Deeplidar(kitti and our own carla dataset) fx=fy=f.
How to set the parameters: 
you can find the following in the clean.hpp
float fcxcy[3];//SET THE CAMERA PARAMETERS  F CX CY BEFORE USE calplanenormal.
int  WINDOWSIZE=15;//SET SEARCH WINDOWSIZE(SUGGEST 15) BEFORE USE calplanenormal.
float Tthrehold;//SET THE threshold (SUGGEST 0.1-0.2)BEFORE USE calplanenormal.
